shared/js/media/video_player.js

```diff
@@ -471,6 +483,20 @@ function VideoPlayer(container) {
     }
   });
 
+  slider.addEventListener('keypress', function(e) {
+    // Currently, we only allow screen reader users to adjust sliders with a
+    // constant step size (there is no page up/down equivalent). In the case
+    // of videos, we make sure that the maximum amount of steps for the entire
+    // duration is 20, or 2 second increments if the duration is less then 40
+    // seconds.
```

Comment:
Added:
    // The standard accessible control for sliders is arrow up/down keys.
    // Our screenreader synthesizes those events on swipe up/down gestures.